unit-tests/rsutils/number/running-average/test-double.py

```diff
@@ -0,0 +1,45 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+from rspy import log, test
+from pyrsutils import running_average
+import random
+
+random.seed()
```

Comment:
Can you add a comment about this line?

Reply:
It seeds the random number generator... if it's not clear, I'll add